### DISCUSSION TITLE
tasmota delays RF messages

### DISCUSSION THREAD

**Discussion by bkbartk** [arendst/Tasmota#11402] [Q&A] [answered]
Hello.

I have a sonoff RF bridge and use it to scan raw data. (rfraw 177)
In Home assistant I found that the trigger is only activated every other time.
I'm sure I'm doing something wrong here.
but in the console log I found, that if I hit the button of my doorbell, every other time nothing gets registered in the console log.
and the other time I get 2 registries. as if it is waiting to combine them to add them to the logging together.
Is someone familiar with this issue or is there a way to prevent this from happening?
thank you.

**Comment by barbudor** [comment, marked as answer]
Hi 
There is a 2 second timer to filter repeated messages
2 identical messages must be at least 2 seconds apart to be considered different
Is that what you see ?
